File: src/kyu_7/switcheroo/python/switcheroo_test.py
import codewars_test as test
from solution import switcheroo

@test.describe("Fixed Tests")
def fixed_tests():
    @test.it('Basic Test Cases')
    def basic_test_cases():
        test.assert_equals(switcheroo("abc"), "bac")
        test.assert_equals(switcheroo('aaabcccbaaa'), 'bbbacccabbb')
        test.assert_equals(switcheroo('ccccc'), 'ccccc')
        test.assert_equals(switcheroo('abababababababab'), 'babababababababa')
        test.assert_equals(switcheroo('aaaaa'), 'bbbbb')

File: src/kyu_7/switcheroo/js/switcherooTest.js
const chai = require("chai");
const switcheroo = require("./switcheroo");
const assert = chai.assert;
chai.config.truncateThreshold=0;

describe("Switcheroo", () => {
    it("Fixed tests", () => {
        assert.strictEqual(switcheroo('abc'), 'bac');
        assert.strictEqual(switcheroo('aaabcccbaaa'), 'bbbacccabbb');
        assert.strictEqual(switcheroo('ccccc'), 'ccccc');
    })
});